ius-release.spec

```diff
@@ -39,12 +42,14 @@ install -D -p -m 0644 %{S:10} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY
 install -D -p -m 0644 %{S:11} %{buildroot}%{_sysconfdir}/yum.repos.d/ius.repo
 install -D -p -m 0644 %{S:12} %{buildroot}%{_sysconfdir}/yum.repos.d/ius-testing.repo
 install -D -p -m 0644 %{S:13} %{buildroot}%{_sysconfdir}/yum.repos.d/ius-archive.repo
+install -Dm0644 %{S:30} %{buildroot}%{_sysconfdir}/mock/ius-6-x86_64.cfg
 %endif
 %if %{defined el7}
 install -D -p -m 0644 %{S:20} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-IUS-7
 install -D -p -m 0644 %{S:21} %{buildroot}%{_sysconfdir}/yum.repos.d/ius.repo
 install -D -p -m 0644 %{S:22} %{buildroot}%{_sysconfdir}/yum.repos.d/ius-testing.repo
 install -D -p -m 0644 %{S:23} %{buildroot}%{_sysconfdir}/yum.repos.d/ius-archive.repo
+install -Dm0644 %{S:31} %{buildroot}%{_sysconfdir}/mock/ius-7-x86_64.cfg
```

Comment:
Minor nitpick, break out these flags like the other install commands for consistency.